.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -63,6 +63,9 @@ body:
         - 3.7
         - 3.8
         - 3.9
+        - 3.10
+        - 3.11
+        - 3.12
```

```suggestion
        - 3.8
        - 3.9
        - 3.10
        - 3.11
```

Then 3.12 in the respective branch

We could also change this to an optional write-in box so we don't have to keep the options up to date

---

That would be better. Let's do that in another PR. though.